utopia-api/src/property-controls/property-controls.ts

```diff
@@ -107,6 +109,13 @@ export interface StyleObjectControlDescription
   defaultValue?: CSSProperties
   placeholder?: CSSProperties
 }
+export interface Vector2ControlDescription extends AbstractBaseControlDescription<'vector2'> {
+  defaultValue?: unknown
```

Comment:
These default values should probably be `[number, number]` / `[number, number, number]` or `Array<number>`

Reply:
avoid Array<number> if possible!

Reply:
fixed!